keyboards/zeal60/keymaps/default/keymap.c

```diff
@@ -0,0 +1,39 @@
+// Default layout for Zeal60
+
+#include "zeal60.h"
```

Comment:
You can replace this with `#include QMK_KEYBOARD_H`

Reply:
fixed